cmd/server-mux.go

```diff
@@ -70,37 +65,35 @@ func longestWord(strings []string) int {
 			maxLen = len(m)
 		}
 	}
-
 	return maxLen
 }
 
 // NewConnMux - creates a new ConnMux instance
 func NewConnMux(c net.Conn) *ConnMux {
-	h1 := longestWord(defaultHTTP1Methods)
-	h2 := longestWord(defaultHTTP2Methods)
-	max := h1
-	if h2 > max {
-		max = h2
+	br := bufio.NewReader(c)
+	bw := bufio.NewWriter(c)
+	return &ConnMux{
+		Conn:    c,
+		bufwr:   bufio.NewReadWriter(br, bw),
```

Comment:
> It means bufwr - bufferred writer and reader.

you know it because you wrote it, but for someone new reading the code bufwr reads like `buffer write`

> It is not a buffered version of Conn.

it is buffered Conn:
```
br := bufio.NewReader(c)
bw := bufio.NewWriter(c)
bufwr: bufio.NewReadWriter(br, bw)
```